run_scratchblocks4.js

```diff
@@ -1,22 +1,64 @@
-mw.hook('wikipage.content').add(function run_scratchblocks() {
-	var version = mw.config.get('wgScratchBlocks4BlockVersion');
+mw.hook("wikipage.content").add(function run_scratchblocks() {
+	var version = mw.config.get("wgScratchBlocks4BlockVersion");
 	var scale = 1;
 	// Note: the weak equality is intentional to allow '2' and 2 both specify sb2
-	if (version == 2 || version[0] == '2') { // to handle '2.0'
-		version = 'scratch2';
+	if (version == 2 || version[0] == "2") {
+		// to handle '2.0'
+		version = "scratch2";
 	} else {
-		version = 'scratch3';
+		version =
+			(version === "scratch3-high-contrast" || version === "scratch3-hc")
```

Comment:
To be consistent with the `version` attribute, `3-hc` is probably the correct code (and `3.0-hc` should also be included as an alias).

```suggestion
			["3-hc", "3.0-hc", "scratch3-high-contrast", "scratch3-hc"].includes(version)
```